Modules/Core/Tests/Unit/DateFieldAutoPopulationTest.php

```
@@ -355,4 +355,32 @@ public function it_filters_numberings_by_current_company_id(): void
         $this->assertCount(1, $otherCompanyGroups, 'Other company should have its document group');
         $this->assertEquals($otherCompanyDocGroup->id, $otherCompanyGroups->first()->id);
     }
+
+    #[Test]
+    #[Group('date-auto-population')]
+    public function it_populates_date_fields(): void
+    {
+        /* arrange */
+        $customer      = Relation::factory()->for($this->company)->customer()->create();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$customer'. (undefined)

(UnusedLocalVariable)